scipy/ndimage/src/nd_image.h

```diff
@@ -34,12 +34,8 @@
 
 #include "Python.h"
 
-#ifndef ND_IMPORT_ARRAY
-#define NO_IMPORT_ARRAY
-#endif
-
+#define PY_ARRAY_UNIQUE_SYMBOL _scipy_ndimage_ARRAY_API
```

Comment:
Might be a good idea to leave a comment explaining how this works, maybe linking to https://docs.scipy.org/doc/numpy/reference/c-api.array.html#importing-the-api.